crates/client/rpc-core/Cargo.toml

```diff
@@ -32,6 +32,7 @@ jsonrpsee = { workspace = true, features = [
 ], default-features = true }
 mp-block = { workspace = true }
 mp-digest-log = { workspace = true }
+mp-transactions = { workspace = true, features = ["serde"] }
```

Comment:
if you do that `serde = ["mp-transactions/serde"]`you don't need to do this `features = ["serde"]`

Reply:
But it look like serde is always imported. So its not an optional feature. So your most likely want `features = ["serde"]` and not `serde = ["mp-transactions/serde"]`

Reply:
Done!